src/runtime/cmd/kata-monitor/README.md

```diff
@@ -0,0 +1,59 @@
+# Kata monitor
+
+## Overview
+`kata-monitor` is a deamon able to collect and expose metrics related to all the Kata Containers workloads running on the same host.
+Once started, it detects all the running Kata Containers runtimes (`containerd-shim-kata-v2`) in the system and exposes few http endpoints to allow the retrieval of the available data.
+The main endpoint is the `/metrics` one which aggregates metrics from all the kata workloads.
+Available metrics include:
+  * Kata runtime metrics
+  * Kata agent metrics
+  * Kata guest OS metrics
+  * Hypervisor metrics
+  * Firecracker metrics
+  * Kata monitor metrics
+
+All the provided metrics are in Prometheus format. While Kata monitor can be used as a standalone daemon on any host running Kata Containers workloads and can be used for retrieving profiling data from the running Kata runtimes, its main expected usage is to be deployed as a Daemonset on a Kubernetes cluster: there Prometheus should scrape the metrics from the kata-monitor endpoints.
```

Comment:
Kata monitor should maybe be kata-monitor here since it refers to the daemon

Reply:
Right, makes sense, changed